modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -121,22 +119,34 @@ def openapi_spec
                 example: '7efd87fc-fac1-4851-a4dd-b9aa2533f57f',
                 schema: { type: :string, format: :uuid }
 
-      response '200', 'Info about a single Supplemental Claim' do
+      veteran_scopes = %w[veteran/SupplementalClaims.read]
+
+      response '200', 'Success' do
         schema '$ref' => '#/components/schemas/scCreateResponse'
 
         let(:id) { FactoryBot.create(:supplemental_claim_v0).id }
 
         it_behaves_like 'rswag example', desc: 'returns a 200 response',
                                          response_wrapper: :normalize_appeal_response,
-                                         scopes:
+                                         scopes: veteran_scopes
+      end
+
+      response '403', 'Forbidden access with a veteran-scoped OAuth token to an unowned Supplemental Claim' do
+        schema '$ref' => '#/components/schemas/errorModel'
+
+        let(:id) { FactoryBot.create(:supplemental_claim_v0, veteran_icn: '1234567890V123456').id }
+
+        it_behaves_like 'rswag example',
+                        desc: 'with a veteran-scoped OAuth token for a Veteran who does not own the Higher-Level Review',
```

Comment:
I think this should be "Supplemental Claim" instead.

Reply:
Thanks, a copy and past fail - fixed now!